src/wasm/cpp/wasm-manifest.h

```diff
@@ -161,4 +161,124 @@ struct std::hash<arcs::Data> {
   }
 };
 
+namespace arcs {
+
+class Info {
+public:
+  // Entities must be copied with arcs::clone_entity(), which will exclude the internal id.
```

Comment:
Can we have a copy constructor that is just a call to `clone_entity` or do we want to enforce that explicit call?

Reply:
I want to enforce it: a) implicit copies are one of the biggest pitfalls of C++, and b) the clone op explicitly does not copy the internal id and I want to make that as obvious as possible.